modules/Bio/EnsEMBL/Taxonomy/PipeConfig/ImportNCBItaxonomy_conf.pm

```diff
@@ -77,7 +77,12 @@ sub default_options {
         'taxdump_file'  => 'taxdump.tar.gz',                        # the filename of the dump
 
         'work_dir'      => $ENV{'HOME'}.'/ncbi_taxonomy_loading',
-    };
+        'copy_service_uri' => "http://production-services.ensembl.org/api/dbcopy/requestjob",
+        'tgt_host'         =>  undef,  
+        'tgt_db_name'      =>  undef, 
+        'src_incl_tables'  => "ncbi_taxa_name,ncbi_taxa_node", 
+        'payload'       => '{ "src_host": "'.$self->o('host').':'.$self->o('port').'", "src_incl_db" : "'.$self->o('ENV', 'USER').'_ncbi_taxonomy'.$self->o('ensembl_release').'", "tgt_host": "'.$self->o('tgt_host').'", "tgt_db_name": "'.  $self->o('tgt_db_name')  . '","src_incl_tables": "'. $self->o('src_incl_tables') .'"'.',"user" : "'.$self->o('ENV', 'USER').'","email": "'.$self->o('ENV', 'USER').'@ebi.ac.uk"}'
```

Comment:
Not sure the payload is all right, you specified 
`"src_incl_db" : $self->o('ENV', 'USER').'_ncbi_taxonomy'.$self->o('ensembl_release').` => db generated is named `ncbi_taxonomy` 
the versioned one is the copied db so this may be the `tgt_db_name` instead. 
and
- No need to specify the include_tables, if none, this will simply be all tables
- You may specify defaults to undef for the following:
        'host'          => undef,
        'port'          => undef,
        'pass'          => undef,
        'user'          => undef,

Reply:
When i execute the taxonomy pipeline  for release 102 it created db with name "{USER}_ncbi_taxonomy_{ENS_VERION}"  eg: vinay_ncbi_taxonomy102. so i used the $self->o('ENV', 'USER').'_ncbi_taxonomy'.$self->o('ensembl_release') 

default 'src_incl_tables'  is  set to "ncbi_taxa_name,ncbi_taxa_node", as db contains hive related tables.
i set to all tables in code and add this tables names in SOP.